go/vt/wrangler/migrater.go

```diff
@@ -543,11 +597,13 @@ func (mi *migrater) cancelMigration(ctx context.Context) {
 		mi.wr.Logger().Errorf("Cancel migration failed:", err)
 	}
 
-	err = mi.forAllUids(func(target *miTarget, uid uint32) error {
-		if _, err := mi.wr.tmc.VReplicationExec(ctx, target.master.Tablet, binlogplayer.StartVReplication(uid)); err != nil {
-			return err
-		}
-		return nil
+	sm := &streamMigrater{mi: mi}
+	sm.cancelMigration(ctx)
+
+	err = mi.forAllTargets(func(target *miTarget) error {
+		query := fmt.Sprintf("update _vt.vreplication set state='Running', message='' where db_name=%s and workflow=%s", encodeString(target.master.DbName()), encodeString(mi.workflow))
```

Comment:
Does it make sense to put in a non-empty message?

Reply:
For now, the message must be empty because if you transition from an error state back to running, the message must be cleared. Due to this, the vplayer always clears the message before it starts to apply changes.